website/models.py

```diff
@@ -251,6 +251,7 @@ class Issue(models.Model):
     modified = models.DateTimeField(auto_now=True)
     is_hidden = models.BooleanField(default=False)
     rewarded = models.PositiveIntegerField(default=0)  # money rewarded by the company
+    viewer = models.ManyToManyField(User, null=True, blank=True, related_name="viewer")
```

Comment:
don't we already have a viewer?

Reply:
@DonnieBLT yup I have replied on it too.
viewer is field to give view access on is_hidden issues by the owner of issue.
there was no other field like this